pymc/sampling_jax.py

```diff
@@ -255,6 +256,8 @@ def sample_blackjax_nuts(
         Include untransformed variables in the posterior samples. Defaults to False.
     chain_method : str, default "parallel"
         Specify how samples should be drawn. The choices include "parallel", and "vectorized".
+    postprocessing_backend : Optional[str]
```

Comment:
I think the numpydoc type hints are something like this:

```suggestion
    postprocessing_backend : str, Optional
```